src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -3486,4 +3488,30 @@ void openapiTest(VertxTestContext context) {
                 context.completeNow();
             });
     }
+
+    @Test
+    void createConsumerWithGeneratedName(VertxTestContext context) {
+        String groupId = "my-group";
+
+        JsonObject json = new JsonObject();
+        json.put("auto.offset.reset", "earliest");
+        json.put("enable.auto.commit", "true");
+        json.put("fetch.min.bytes", "100");
```

Comment:
I don't think this test needs other parameters for consumer creation. Let's keep it simple.